Doc/library/typing.rst

```diff
@@ -1976,6 +1976,11 @@ Other concrete types
 
    .. versionadded:: 3.5.2
 
+   .. deprecated:: 3.11
+      Python 2 is no longer supported at runtime, and is also no longer
+      supported by most type checkers. Users should therefore now use
```

Comment:
"at runtime" is a little confusing, especially when combined with the "deprecated in Python 3.11"
```suggestion
   .. deprecated:: 3.11
      Python 2 is no longer supported and most type checkers no longer
      support type checking Python 2 code either. Users should now use
```

Reply:
Nice!